tests/foreman/api/test_docker.py

```diff
@@ -1083,146 +1083,77 @@ def setUpClass(cls):
             url=EXTERNAL_DOCKER_URL,
         ).create()
 
-    def setUp(self):
-        """Generate default ``DockerHubContainer`` keyword arguments.
-
-        Generate a dict of arguments that can be used when instantiating a
-        ``DockerHubContainer`` and save it as ``self.dhc_kwargs``.
-
-        """
-        self.container_name = gen_string('alphanumeric')
-        self.dhc_kwargs = {
```

Comment:
I think that is fine, but just use setUpClass instead. I think that is a nice style to prepare init section for future